_posts/2018-08-29-bitcoin-abc-and-the-block-size-limit.md

```diff
@@ -0,0 +1,87 @@
+---
+layout: post
+title: Bitcoin ABC and the Block Size Limit
+subtitle: Our goal is to enable larger blocks by doing the necessary technical work
+multiLangId: 2018-08-29-bitcoin-abc-and-the-block-size-limit
```

Comment:
8-29 to 9-7 (here and filename), so this publication will occur after the sharding article